packages/vscode-extension/src/project/applicationSession.ts

```diff
@@ -652,6 +678,32 @@ export class ApplicationSession implements Disposable {
     this.stateManager.updateState({ logCounter: 0 });
   }
 
+  public resetHomeTarget() {
+    this.navigationHomeTarget = undefined;
+  }
```

Comment:
I don't like this. I feel like this is `ApplicationSession`'s internal state, and shouldn't be controlled by something outside of it.

We only call this when first starting or reloading the app.
In either case we either:
- create a new `ApplicationSession` which will clear this anyway
- reload JS
Maybe we should simply clear this in `reloadJS`?